src/gettingstarted/own-code/create-project.md

```diff
@@ -0,0 +1,57 @@
+# Import your own code
+
+## Create a new project
+
+With the Platform.sh CLI now installed and configured to communicate with your projects, you can create a new project from the command line and connect it to your application.
```

Comment:
"configured to communicate with your account".  The very first command is communicating with not-a-project-yet. :smile: